assets-windows/installer.nsi

```diff
@@ -0,0 +1,79 @@
+!include "MUI2.nsh"

```

Comment:
Can we move these files into `assets/windows/`? This way we can then move other things like the MacOS images for the dmg into `assets/macos/`